credoai/lens/lens.py

```diff
@@ -184,6 +183,8 @@ def send_to_governance(self, overwrite_governance=False):
         """
         evidence = self.get_evidence()
         if self.gov:
+            if self.model:
```

Comment:
If model exists, send information about it. Not sure what to do when no model is provided!